src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="TESTHOST/testfqdn.test.corefx.net"
+principal2="TESTHTTP"
+krb_user="krb_user"
+password="password"
```

Comment:
We couldn't think of a way to create a random password that the test case also would be aware of. @stephentoub, if the test case is always running elevated, we can store a random password in a well-known file which requires root access for reading. But if we expect the 3rd party dev experience running tests outside CI to run non-elevated, this would be an issue.